ironfish-cli/src/command.ts

```diff
@@ -127,6 +130,11 @@ export abstract class IronfishCommand extends Command {
       configOverrides.rpcTcpSecure = rpcTcpSecureFlag
     }
 
+    const rpcConnectTlsFlag = getFlag(flags, RpcUseTlsFlagKey)
```

Comment:
nit: This is small issue but it doesn't match the flag name like all the other flags here